### ISSUE TITLE
supervised or unsupervised

### ISSUE THREAD

**Issue by ZahraDalvand** [3dv-casia/PolyRoom#11]
Hello,

Thank you for sharing your great work on PolyRoom. I have a question regarding the training setup.
From the repository, I noticed that ground-truth masks are used for training Mask R-CNN and the Transformer components.

Could you please clarify whether PolyRoom is trained in a **fully supervised** manner using annotated masks, or does it also involve any **unsupervised / self-supervised** components?

This information would help me better understand the learning paradigm of your approach.

Thanks in advance for your time and support!

Best regards

**Comment by PaulLiuYZ** [comment]
It's trained in a fully supervised manner.
